.circleci/config.yml

```diff
@@ -1196,24 +1214,24 @@ jobs:
       compile_only:
         type: integer
         default: 0
-      nodejs_version:
+      image:
         type: string
-        default: current
+        default: cimg/node:current
```

Comment:
You could put this change in a separate small PR, along with the `t_ext` name change. It would be quicker to go through review and would leave less noise in the main PR, which is not ready yet anyway.

Reply:
Done in: https://github.com/ethereum/solidity/pull/13982